metaflow/plugins/pypi/pip.py

```diff
@@ -150,6 +151,85 @@ def _format(dl_info):
                     res["hash"] = vcs_info["commit_id"]
                 return res
 
+            from packaging.requirements import Requirement
```

Comment:
note implicit dependency on packaging. this is at least limited to only the solve step, but still not desirable

Reply:
packaging is already vendored.